starport/templates/app/templates/.github/workflows/build.yml.plush

```diff
@@ -0,0 +1,60 @@
+# This is a basic workflow that is manually triggered
+
+name: <%= AppName %>
+
+# Controls when the action will run. Workflow runs when manually triggered using the UI
+# or API.
+on: [push, pull_request]
+
+# This workflow makes x86_64 binaries for mac, windows, and linux.
+
+
+jobs:
+  mac-windows:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        targetos: [windows, darwin]
+    name: <%= AppName %> for ${{ matrix.targetos }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup go
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.15
+        env:
+          GOOS: ${{ matrix.targetos }}
+      - name: Compile
+        run: |
+          go mod vendor
+          mkdir build
+          go build -mod=readonly -o build/<%= BinaryNamePrefix %>d ./cmd/<%= BinaryNamePrefix %>d
+          go build -mod=readonly -o build/<%= BinaryNamePrefix %>cli ./cmd/<%= BinaryNamePrefix %>cli
+      - uses: actions/upload-artifact@v2
```

Comment:
I wonder where these binaries are uploaded to or where they're downloaded at with `actions/download-artifact`. Should we think about using `goreleaser/goreleaser` to actually create Github releases?

Reply:
I think it's premature to make releases for a fresh starport chain, though we could include it in the template so that a release is built and released ever time there is a new tag.  

The binaries go here:

https://github.com/faddat/clay/actions/runs/262801322

They're basically provided for visibility and ease of use, and they’re built with each push/pull as "artifacts" of the CI job run.